lib/VM/JSLib/GlobalObject.cpp

```diff
@@ -754,12 +754,15 @@ void initGlobalObject(Runtime *runtime, const JSLibFlags &jsLibFlags) {
 #ifdef HERMES_PLATFORM_INTL
   // Define the global Intl object
   // TODO T65916424: Consider how we can move this somewhere more modular.
-  runtime->ignoreAllocationFailure(JSObject::defineOwnProperty(
-      runtime->getGlobal(),
-      runtime,
-      Predefined::getSymbolID(Predefined::Intl),
-      normalDPF,
-      intl::createIntlObject(runtime)));
+  
+  if (LLVM_UNLIKELY(runtime->hasES6Intl())) {
```

Comment:
Hey @mganandraj, I'm a little confused by the purpose of having this runtime config. 

If Hermes would be compiled with the Intl flag on for OSS, then the binary size has been already increased. Does unset this runtime config help with any savings?